ESCPOS_NET/Printers/USBPrinter.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.IO;
+
+namespace ESCPOS_NET.Printers
+{
+    public class USBPrinter : BasePrinter
+    {
+        private readonly FileStream _rfile;
+        private readonly FileStream _wfile;
+        public USBPrinter(string usbPath)
+            : base()
+        {
+            //keeping separate file streams performs better
+            //while using 1 file stream printers were having intermittent issues while printing
+            //your milege may vary
+            _rfile = File.Open(usbPath, FileMode.Open, FileAccess.Read);
```

Comment:
I got an error and cannot findout how to resolve, have you got this error too?
`FileStream was asked to open a device that was not a file. For support for devices like 'com1:' or 'lpt1:', call CreateFile, then use the FileStream constructors that take an OS handle as an IntPtr.`